src/components/Footer/Footer.spec.js

```diff
@@ -15,7 +15,8 @@ test("renders NotFound correctly and matches snapshot", async () => {
   expect(asFragment()).toMatchSnapshot();
 
   // Content test
-  expect(getByText(/Copyright 2020. All Rights Reserved/)).toBeInTheDocument();
+  expect(getByText(/Copyright 2020./)).toBeInTheDocument();
+  expect(getByText(/All Rights Reserved/)).toBeInTheDocument();
```

Comment:
Why do we need to split this? In all designs these 2 lines should be together and if the test case fails it means that the design is wrong.

Reply:
I wrapped these two sections in separate spans in commit https://github.com/poulami-saha/Global-Code-Queens-MyTeam-Website/pull/39/commits/edeae5a4ffa76a08dda1ff7008d47547c7036026 because at narrow desktop sizes (like 960-980px or so) it was wrapping only the word "reserved" and it looked terrible. This spec is updated to handle the text being in separate spans. 
https://github.com/poulami-saha/Global-Code-Queens-MyTeam-Website/blob/e78a31a539e40ca1fbdea1f2a25fc9371d9d5cc8/src/components/Footer/Footer.jsx#L44

Reply:
I found this solution to set preferred line wrapping points on this [StackOverflow question](https://stackoverflow.com/questions/5392853/denoting-a-preferred-place-for-a-line-break).